front50-core/src/main/groovy/com/netflix/spinnaker/front50/model/delivery/DeliveryEnvironment.java

```diff
@@ -0,0 +1,28 @@
+package com.netflix.spinnaker.front50.model.delivery;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import java.util.List;
+import java.util.Map;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class DeliveryEnvironment {
+  String name;
+  List<Map<String,Object>> infrastructure; //todo eb: let's type this.
```

Comment:
We should maybe type it as we go. Add definitions here as we support them in Keel. I'm hesitant to suggest sharing a definition but maybe that would be appropriate for once.